src/plugins/discover/public/application/components/discover_grid/discover_grid.tsx

```diff
@@ -258,7 +263,13 @@ export const DiscoverGrid = ({
         isDarkMode: services.uiSettings.get('theme:darkMode'),
       }}
     >
-      <>
+      <span
+        data-test-subj="discoverDocTable"
+        data-render-complete={isLoading ? 'false' : 'true'}
+        data-shared-item=""
```

Comment:
@majagrubic here's data `data-shared-item`, thought it's cleaner to migrate all those `data-*`.